documentation/source/user_section/tutorials/spinalcord-segmentation/sct_propseg-example-t1.rst

```diff
@@ -0,0 +1,44 @@
+Hands-on Example: T1
+####################
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t1_image.png
+  :align: right
+  :figwidth: 8%
+
+  t1.nii.gz
+
+Next, we will navigate to the T1 directory and verify that it contains a single T1-weighted image. If you are still in the T2 directory from the previous section, this can be done as follows:
+
+.. code:: sh
+
+   cd ../t1
+   ls
+   # Output
+   # t1.nii.gz
```

Comment:
oopsi 😬
~~~
cd ../
ls
# Output: t2
~~~

t1/ folder is missing.

Reply:
Aha, this is a silly issue. I linked to the registration dataset, not the segmentation dataset. :sweat_smile: